public/app/features/dashboard-scene/saving/useSaveDashboard.ts

```diff
@@ -28,6 +28,7 @@ export function useSaveDashboard(isCopy = false) {
           message: options.message,
           overwrite: options.overwrite,
           showErrorAlert: false,
+          k8s: undefined, // TODO?  pass the original metadata
```

Comment:
@dprokop -- where does the dashboard model, metadata exist in scenes?  In the non-scenses flavor we set it here https://github.com/grafana/grafana/pull/89598/files#diff-2d1715bd81b5570ae881b8c23cdbea860d6c1bc2944a506866d5a07921755793R29